include/networkit/graph/GraphBuilder.hpp

```diff
@@ -166,7 +166,7 @@ class GraphBuilder {
     /**
      * Generates a Graph instance. The graph builder will be reseted at the end.
      */
-    Graph toGraph(bool autoCompleteEdges, bool parallel = false);
+    Graph toGraph(bool parallel = false);
```

Comment:
We should probably not change the signature of the function silently. Instead, let it still take the autoCompleteEdges parameter but throw an exception if it is set to false. Also deprecate the function.

We can add another `Graph completeGraph()` function that does not take the parameter. (And `toGraph()` can simply call `completeGraph()`.